bower.json

```diff
@@ -41,7 +41,8 @@
     "ngInfiniteScroll": "1.0.0",
     "pnotify": "1.3.1",
     "sticky": "*",
-    "swagger-ui": "wordnik/swagger-ui#v2.0.24"
+    "swagger-ui": "wordnik/swagger-ui#v2.0.24",
+    "smart-app-banner": "*"
```

Comment:
@kelsin could you specify a specific version for the package? We don't want a future update to break the site